.circleci/config.yml

```diff
@@ -93,15 +46,78 @@ anchor_7: &setup_bazel_remote_execution
       sudo bash -c "echo -e 'build --config=remote' >> .bazelrc.user";
       echo "Reading from remote cache for bazel remote jobs.";
 
+# Executor Definitions
+executors:
+  action-executor:
+    docker:
+      - image: circleci/node:12.1
+    working_directory: ~/ng
+
+  test-executor:
+    parameters:
+      nodeversion:
+        type: string
+        default: "12.1"
+    docker:
+      - image: circleci/node:<< parameters.nodeversion >>-browsers
```

Comment:
Is it possible to use an anchor as a parameter default? Then we could have `var_5: 12.1` and use it in both executors. This way it would be easier for us to remember to change both together.